amf-client/shared/src/test/resources/production/oas-complex-example/api.resolved.expanded.jsonld

```diff
@@ -1572,15 +1546,15 @@
                         ],
                         "http://a.ml/vocabularies/document-source-maps#lexical": [
                           {
-                            "@id": "file://amf-client/shared/src/test/resources/production/oas-complex-example/spec/swagger.json#/web-api/end-points/%2Fpets/post/200/source-map/lexical/element_2",
+                            "@id": "file://amf-client/shared/src/test/resources/production/oas-complex-example/spec/swagger.json#/web-api/end-points/%2Fpets/post/200/source-map/lexical/element_1",
                             "http://a.ml/vocabularies/document-source-maps#element": [
                               {
-                                "@value": "http://a.ml/vocabularies/core#name"
```

Comment:
It appears lexical info of OAS Response names is being lost, not sure if its worth taking a deeper look.